### ISSUE TITLE
error with paper #417

### ISSUE THREAD

**Issue by AKtomik** [hotpocket184/skript-advancements#23]
when the server start, this error happend and skript-pack does not work correctly :

```
[02:57:16] [Server thread/ERROR]: [global] TIMING_STACK_CORRUPTION - Report this to the plugin Skript (Look for errors above this in the logs) (TimingIdentifier{id=Skript:Additions/Quests/Commands.sk: line 43} did not stopTiming)
java.lang.Throwable: null
at co.aikar.timings.TimingHandler.popTimingStack(TimingHandler.java:132) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-417]
at co.aikar.timings.TimingHandler.stopTiming(TimingHandler.java:117) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-417]
at co.aikar.timings.TimingHandler.stopTimingIfSync(TimingHandler.java:96) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-417]
at co.aikar.timings.TimingHandler.close(TimingHandler.java:202) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-417]
at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftTask.run(CraftTask.java:99) ~[paper-1.19.3.jar:git-Paper-417]
at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1473) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:447) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1397) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1173) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-417]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
[02:57:16] [Server thread/WARN]: [Skript] Task #400 for Skript v2.6.4 generated an exception
java.lang.NoSuchMethodError: 'java.lang.String org.bukkit.UnsafeValues.getTranslationKey(org.bukkit.Material)'
at me.hotpocket.skriptadvancements.utils.creation.TempAdvancement.getTexture(TempAdvancement.java:139) ~[skriptadvancements-1.5.3.jar:?]
at me.hotpocket.skriptadvancements.utils.creation.TempAdvancement.build(TempAdvancement.java:169) ~[skriptadvancements-1.5.3.jar:?]
at me.hotpocket.skriptadvancements.elements.effects.creation.EffBuildAdvancement.execute(EffBuildAdvancement.java:31) ~[skriptadvancements-1.5.3.jar:?]
at ch.njol.skript.lang.Effect.run(Effect.java:50) ~[Skript-2.6.4.jar:?]
at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61) ~[Skript-2.6.4.jar:?]
at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89) ~[Skript-2.6.4.jar:?]
at ch.njol.skript.effects.Delay$1.run(Delay.java:109) ~[Skript-2.6.4.jar:?]
at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.19.3.jar:git-Paper-417]
at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1473) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:447) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1397) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1173) ~[paper-1.19.3.jar:git-Paper-417]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-417]
at java.lang.Thread.run(Thread.java:833) ~[?:?] 
```
This happend since i update the server version to paper#417 (before, that working perfectly)
If you need more detail, tell me.

**Comment by hotpocket184** [comment]
I see the issue. I will fix it soon. I will most likely just attempt to make my own translation key getter. Thank you!

**Comment by hotpocket184** [comment]
Fixed in 1.5.4
